src/lcd.c

```diff
@@ -168,10 +164,17 @@ void LCD_SendWord(uint8_t type, uint8_t word){
     HC595_Write(LCD_DISABLE | type | low);
 }
 
-void LCD_SendASCIIChar(char* character){
+void LCD_SendASCIIChar(const char* character){
     LCD_SendWord(LCD_DATA, (uint8_t)(*character));
 }
 
+void LCD_SendText(char* text){
+    const char* ch = text;
+    while(*ch){
```

Comment:
Gonna keep this function like it is because it doesn't actually result in undefined behavior. In fact, this is how long strings are displayed when scrolling is enabled.